tests/virtualization/universal/patch_guests.pm

```diff
@@ -33,7 +33,8 @@ sub run {
     record_info "BEFORE", "This phase is BEFORE the patching";
     foreach my $guest (keys %virt_autotest::common::guests) {
         record_info "$guest", "Probing the guest, adding test repositories and patching the system";
-        ensure_online($guest, skip_ping => (is_hyperv_virtualization || is_vmware_virtualization));
+        ensure_online($guest, skip_ping => (is_hyperv_virtualization || is_vmware_virtualization))
+          foreach (keys %virt_autotest::common::guests);
```

Comment:
We already loop over `%virt_autotest::common::guests` on line 34.

Reply:
I will remove line 34 and rerun the VR, thanks.